src/gui_common/art_gallery/GalleryCard.cs

```diff
@@ -22,7 +22,7 @@ public class GalleryCard : Button
     ///   If this is true, this item can be featured in slideshow.
     /// </summary>
     [Export]
-    public bool CanBeSlideshown { get; set; } = true;
+    public bool CanBeShownInASlideshow { get; set; } = true;
```

Comment:
Is this used in any scenes (as this is an export variable)? Where this might need to be also renamed.

Reply:
Currently not.